src/pages/guides/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Guides
+metaTitle: Create Spl Token
+description: Learn how to create an Spl Token on the Solana blockchain.
+---
+
+![image info](./assets/banners/touch-screen.jpg)
+
+The guides in this section are **designed to help new Solana developers start their journey** by providing step-by-step instructions and practical examples. Recognizing the complexity of learning blockchain technology, these guides focus on teaching key concepts with a structured learning path that gradually introduces developers to the Solana ecosystem. They cover everything from the basics of blockchain and Solana's unique features to advanced topics such as smart contract development and optimization techniques.
+
+By incorporating a variety of learning resources, including tutorials and code samples, **the guides are accessible to all levels of expertise**. They emphasize hands-on experience, encouraging developers to build and deploy their own applications since this practical approach not only reinforces theoretical knowledge, but also boosts confidence and competence. Additionally, the guides are regularly updated to reflect the latest developments in the Solana ecosystem, ensuring that new developers have access to current information and best practices.
```

Comment:
I like this is it much more concise then the last rev.

wdyt about removing "but also boosts confidence and competence."

and _possibly_ "ensuring that new developers have access to current information and best practices."

Those two still seem slightly extra sounding to me, wdyt?